COPY/usr/bin/manageiq-db-ready

```diff
@@ -25,8 +27,11 @@ end
 def manageiq_db_ready?(db_yaml, env = "production")
   host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
   password = ManageIQ::Password.try_decrypt(password)
+  host_arg = host.to_s.strip.empty?     ? "--host=#{host} " : ""  # If not present, let postgres decide to use localhost/socket
+  port_arg = port.to_s.strip.empty?     ? "--port=#{port} " : ""
+  pass_arg = password.to_s.strip.empty? ? "PGPASSWORD=#{password} " : ""
```

Comment:
I guess these should just be `nil?` ... we shouldn't handle empty or blank strings.  Let me fix that.

Reply:
This is done now.